domain/src/main/kotlin/no/nav/su/se/bakover/domain/klage/VilkårsvurdertKlage.kt

```diff
@@ -108,20 +105,22 @@ sealed class VilkårsvurdertKlage : Klage() {
         }
     }
 
-    data class Utfylt private constructor(
-        override val id: UUID,
-        override val opprettet: Tidspunkt,
-        override val sakId: UUID,
-        override val saksnummer: Saksnummer,
-        override val fnr: Fnr,
-        override val journalpostId: JournalpostId,
-        override val oppgaveId: OppgaveId,
-        override val saksbehandler: NavIdentBruker.Saksbehandler,
-        override val vilkårsvurderinger: VilkårsvurderingerTilKlage.Utfylt,
-        override val vurderinger: VurderingerTilKlage?,
-        override val attesteringer: Attesteringshistorikk,
-        override val datoKlageMottatt: LocalDate,
-    ) : VilkårsvurdertKlage() {
+    /**
+     * Denne tilstanden representerer en klage når alle vilkårsvurderingene er blitt fylt ut, og ikke har blitt bekreftet
+     */
+    sealed class Utfylt : VilkårsvurdertKlage {
+        abstract override val id: UUID
```

Comment:
Disse kan skippes, men du har de kanskje med for oversiktens skyld?

Reply:
Ja, litt enklere å finne fram ting når man ser når nye klasser starter